.changeset/cyan-walls-sell.md

```diff
@@ -0,0 +1,5 @@
+---
+'@leafygreen-ui/info-sprinkle': major
```

Comment:
Re:Slack thread, since this was broken before, this can be considered a bug fix
![](https://files.slack.com/files-pri/T01C4Q4H3CL-F09FN3RK11B/screenshot_2025-09-17_at_11.07.50___am.png)

Reply:
I updated - it broke one of our tests so i'm not entirely sure this is correct but happy to go forward with it as a patch if you think that's best